src/Components/Detailspage.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { getCompanyDetails } from '../redux/CompaniesList';
+import '../../styles/Detailspage.css';
+
```

Comment:
- [ ] Kindly make sure you fix the above path error as right now it shows an error when a user want to run your application. This should be this `import '../styles/Detailspage.css';` insted of `import '../../styles/Detailspage.css';`

Reply:
@tresorsawasawa If I change it according to your suggestion then it occurs errors in the project.